src/hooks/useActions.js

```diff
@@ -13,8 +13,8 @@ import { useDisputeFees } from './useDispute'
 import env from '../environment'
 
 import { VOTE_YEA } from '../constants'
```

Comment:
Should we use `@` for all this cases?

Reply:
ah yeah, going to change it in the future while we are working and we see it since it takes a lot of time now heheh